README.md

```diff
@@ -0,0 +1,273 @@
+<p align="center">
+    <img alt="Plone Logo" width="200px" src="https://raw.githubusercontent.com/plone/.github/main/plone-logo.png">
+</p>
+
+<h1 align="center">plone.volto</h1>
+
+<div align="center">
+
+[![PyPI](https://img.shields.io/pypi/v/plone.volto)](https://pypi.org/project/plone.volto/)
+[![PyPI - Python Version](https://img.shields.io/pypi/pyversions/plone.volto)](https://pypi.org/project/plone.volto/)
+[![PyPI - Wheel](https://img.shields.io/pypi/wheel/plone.volto)](https://pypi.org/project/plone.volto/)
+[![PyPI - License](https://img.shields.io/pypi/l/plone.volto)](https://pypi.org/project/plone.volto/)
+[![PyPI - Status](https://img.shields.io/pypi/status/plone.volto)](https://pypi.org/project/plone.volto/)
+
+
+[![PyPI - Plone Versions](https://img.shields.io/pypi/frameworkversions/plone/plone.volto)](https://pypi.org/project/plone.volto/)
+
+[![Meta](https://github.com/plone/plone.volto/actions/workflows/meta.yml/badge.svg)](https://github.com/plone/plone.volto/actions/workflows/meta.yml)
+![Code Style](https://img.shields.io/badge/Code%20Style-Black-000000)
+
+[![GitHub contributors](https://img.shields.io/github/contributors/plone/plone.volto)](https://github.com/plone/plone.volto)
+[![GitHub Repo stars](https://img.shields.io/github/stars/plone/plone.volto?style=social)](https://github.com/plone/plone.volto)
+
+</div>
+
+**plone.volto** configures [Plone](https://plone.org/) to work with [Volto](https://www.npmjs.com/package/@plone/volto), the default frontend Plone since version 6.0.
```

Comment:
```suggestion
`plone.volto` configures [Plone](https://plone.org/) to work with [Volto](https://www.npmjs.com/package/@plone/volto), the default frontend of Plone since version 6.0.
```